ClientApp/viking-vault/src/UserDashboard/ProfilePage.css

```diff
@@ -1,13 +1,13 @@
 .ProfilePage 
 {
     position: absolute;
-    background-color: rgba(223, 219, 219, 0.945);
-    margin-top: 5%;
-    margin-left: 20%;
-    width: 70%;
+    background-color:white;
+    margin-top: 15px;
+    margin-left: 18%;
```

Comment:
Did you add this to center the component? If so then use `margin: auto`

Reply:
Same explanation as the one at the bottom of the page